### ISSUE TITLE
Alt + Ctrl + Shift + Arrow(s) is not working in HTML Table

### ISSUE THREAD

**Issue by afroz-al** [FreedomScientific/VFO-standards-support#522]
### Summary

When a table is wrapped with some  content editable, divs, JAWS fails to recognize this as table 
and  Alt + Ctrl + Shift + Arrows doesn't work.
When we remove this contenteditable divs, it starts to work.

### Expected result

> Alt + Ctrl + Shift + Arrows should move the focus to the top of column (up arrow), button of column (down arrow), beginning of row (left arrow), or end of row (right arrow)

### Actual result

>  JAWS saying cannot move to bottom of column or cannot move to end of row

### Example

> Not Working demo: https://codepen.io/afroz-al/pen/vYgqYEx
   Working demo: https://codepen.io/afroz-al/pen/bGgPGdM

#### JAWS version and build number

JAWS Version 2021.2103.174 ILM
 
#### Operating System and version
Windows 10 Enterprise

#### Browser and version:

Chrome 90
